packages/eslint-plugin/README.md

```diff
@@ -55,13 +55,12 @@ You can also enable all the recommended rules for our plugin. Add `plugin:@types
 
 ### Recommended Configs
 
-You can also use [`eslint:recommended`](https://eslint.org/docs/rules/) (the set of rules which are recommended for all projects by the ESLint Team) with this plugin. As noted in the root README, not all ESLint core rules are compatible with TypeScript, so you need to add both `eslint:recommended` and `plugin:@typescript-eslint/eslint-recommended` (which will adjust the one from ESLint appropriately for TypeScript) to your config:
```

Comment:
I hope my understanding is correct, but I'm happy to be corrected. I just saw the removals in https://github.com/typescript-eslint/typescript-eslint/pull/2001/files#diff-810ec650811df78f6b2eb19fb38ec0d1